Src/Support/CommonPackageInfo.xml

```diff
@@ -0,0 +1,12 @@
+<Project>
+  <PropertyGroup>
+    <PackageVersion>1.25.0</PackageVersion>
+	<Authors>Google Inc.</Authors>
+	<Copyright>Copyright 2017 Google Inc.</Copyright>
```

Comment:
google-api-dotnet-client packages have always had a copyright nupack entry. Yes, I would suggest adding it for cloud.